### DISCUSSION TITLE
添加Polyline问题

### DISCUSSION THREAD

**Discussion by JohnnyRRR** [zouyaoji/vue-cesium#134] [Q&A]
请问如何自定义线段样式？比如根据一组经纬度坐标绘制一段铁丝网

**Comment by zouyaoji** [comment]
> 请问如何自定义线段样式？比如根据一组经纬度坐标绘制一段铁丝网

是说的vc-polyline组件吗？
@JohnnyRRR

**Reply by JohnnyRRR** [reply]
是的，不过我已经查阅资料解决了，使用的自定义的材质绘制的线，感谢！

**Reply by zouyaoji** [reply]
ok 下次可以@一下
